sense.py

```diff
@@ -8,6 +8,25 @@
 import matplotlib.pyplot as plt
 import cv2
 
+class CSTransform(object):
+
+    def __init__(self, compression_factor, img_shp):
+
+        self.rng = np.random.default_rng(seed=21) #Set RNG for repeatble results
```

Comment:
Make RNG seed an object attribute that can be passed when creating transform

Reply:
I think it may be better to keep the seed hard-coded just so that we don't have to worry about inadvertently changing the seed between runs.